h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -136,6 +140,15 @@ protected ModelMetricsListSchemaV3 schema(int version) {
             "specifying a Compact output format will produce a per-feature contribution", values = {"Original", "Compact"}, json = false)
     public Model.Contributions.ContributionsOutputFormat predict_contributions_output_format;
 
+    @API(help = "Only for predict_contributions function - sort Shapley values and return top_n highest (optional)", json = false)
+    public int top_n;
+
+    @API(help = "Only for predict_contributions function - sort Shapley values and return top_bottom_n lowest (optional)", json = false)
+    public int top_bottom_n;
+
+    @API(help = "Only for predict_contributions function - sort absolute Shapley values (optional)", json = false)
+    public boolean abs;
```

Comment:
I think this is missing a verb, how about "compare_abs" - is there a precedence of calling this? this must be used somewhere, not necessarily just for contributions

Reply:
I'll try to find similar atribute

Reply:
I am not aware of the similar attribute in h2o-3. 

- The absolute value in python is `abs(number)` -> that is why I use "abs_val" for both Python and R API
- In numpy it is `np.absolute(x)`
- In R it is `abs(value)`